src/Extensions/PageExtension.php

```diff
@@ -44,12 +46,11 @@ public function IsGetSiteControlEnabledOnPageLevel(): bool
         if ($this->getOwner()->hasMethod('IsGetSiteControlEnabledOnPageLevelOverride')) {
             return $this->getOwner()->hasMethod('IsGetSiteControlEnabledOnPageLevelOverride');
         }
-        $excluded = Config::inst()->get(PageExtension::class, 'page_classes_excluded_from_get_site_control');
-        foreach($excluded as $className) {
-            if($this instanceof $className) {
-                return false;
-            }
-        }
-        return true;
+
+        return in_array(
+            $this->getOwner()->ClassName, 
+            Config::inst()->get(PageExtension::class, 'page_classes_excluded_from_get_site_control'), 
+            true
+        );
```

Comment:
It would be better to use the `instanceof` because of inheritance?

Reply:
I don't understand what you mean sorry.

Reply:
If a page extends ErrorPage then you want that to be excluded automatically as well, hence we use instanceof rather than in_array.  Please fix.